src/main/java/com/puppycrawl/tools/checkstyle/DefaultLogger.java

```diff
@@ -159,29 +159,29 @@ public void addError(AuditEvent event) {
     @Override
     public void addException(AuditEvent event, Throwable throwable) {
         synchronized (errorWriter) {
-            final LocalizedMessage addExceptionMessage = new LocalizedMessage(1,
+            final Violation addExceptionMessage = new Violation(1,
                 Definitions.CHECKSTYLE_BUNDLE, ADD_EXCEPTION_MESSAGE,
                 new String[] {event.getFileName()}, null,
-                LocalizedMessage.class, null);
+                Violation.class, null);
             errorWriter.println(addExceptionMessage.getMessage());
             throwable.printStackTrace(errorWriter);
         }
     }
 
     @Override
     public void auditStarted(AuditEvent event) {
-        final LocalizedMessage auditStartMessage = new LocalizedMessage(1,
+        final Violation auditStartMessage = new Violation(1,
             Definitions.CHECKSTYLE_BUNDLE, AUDIT_STARTED_MESSAGE, null, null,
```

Comment:
@romani Here is an example of this not being a violation.

Reply:
yes, good catch!!

Looks like we need to keep LocalizedMessage class for cases like this.
So we need to 2 classes LocalizedMessage for Loggers and Violation for Checks.
LocalizedMessage should have no fields for column, line, ... tokenType, ... .
we can make separate issue on this.

Right now this Violation class might looks a bit weird in context of Loggers, but look at arguments they mimic empty violation. After class is created we immediately convert to String by
https://github.com/checkstyle/checkstyle/blob/f868b02a37191ca662bb919f44353b55897793b0/src/main/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessage.java#L454

So I am ok with such update (separate issue to create LocalizedMessage class that do a localization for string values).

Reply:
Sir, can you make a separate issue, I can start working on it right away.

Reply:
https://github.com/checkstyle/checkstyle/issues/9794